doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -671,11 +671,18 @@ where syntactic conflicts arise:
     `0` multiplied by the variable `xff`.
   * The floating-point literal expression `1e10` could be interpreted as the numeric literal `1` multiplied
     by the variable `e10`, and similarly with the equivalent `E` form.
+  * The 32-bit floating-point literal expression `1.5f22` could be interpreted as the numeric literal
+    `1.5` multiplied by the variable `f22`.
 
-In both cases, we resolve the ambiguity in favor of interpretation as numeric literals:
+In all cases, we resolve the ambiguity in favor of interpretation as numeric literals:
 
   * Expressions starting with `0x` are always hexadecimal literals.
   * Expressions starting with a numeric literal followed by `e` or `E` are always floating-point literals.
+  * Expressions starting with a numeric literal followed by `f` are always 32-bit floating-point literals.
+
+!!! note
+    The case `1.5F22` is interpreted as the numeric literal `1.5` multiplied by the variable `F22`, because
+    `1.5F22` is not a valid numeric literal representation.
```

Comment:
This note is a bit unclear—maybe explicitly mention the case of the letter `F`?

Reply:
Hi @StefanKarpinski 

Do you mean, something like this:

* Expressions starting with a numeric literal followed by `f` are always 32-bit floating-point literals. On the other hand, expressions starting with a numeric literal followed by `F` are interpreted as a numerical literal multiplied by a variable. Hence, `1.5F22` is equal to `1.5 * F22`.

And remove the note?

Reply:
Yes, that sounds good.